mod/messages/start.php

```diff
@@ -407,7 +402,7 @@ function messages_get_unread($user_guid = 0, $limit = null, $offset = 0, $count
 		'distinct' => false,
 	);
 
-	return elgg_get_entities_from_metadata($options);
+	return elgg_get_entities($options);
```

Comment:
This change was already possible; we don't use any `elgg_get_entities_from_metadata` options.